### PR TITLE
New version: Zoom.Zoom version 5.15.3.18551

### PR DESCRIPTION
**Pull request by mivsvit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112031&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  InstallModes:
+  - interactive
+  - silent
+  InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: /silent
+  ProductCode: ZoomUMX
+  AppsAndFeaturesEntries:
+  - Publisher: Zoom Video Communications, Inc.
+    DisplayVersion: 5.15.2 (18096)
```
Comment (CoolPlayLin) [inline manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml:62]:
```suggestion
    DisplayVersion: 5.15.2.18096
```

File: manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
```
Comment (CoolPlayLin) [inline manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml:46]:
```suggestion
    DisplayVersion: 5.15.2.18096
```

File: manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  - interactive
+  - silent
+  InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: /silent
+  ProductCode: ZoomUMX
+  AppsAndFeaturesEntries:
+  - DisplayName: Zoom(32bit)
+    Publisher: Zoom Video Communications, Inc.
+    DisplayVersion: 5.15.2 (18096)
```
Comment (CoolPlayLin) [inline manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml:30]:
```suggestion
    DisplayVersion: 5.15.2.18096
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

**Comment by BrandonWanHuanSheng** [comment]
I noticed Wingetcreate and Komac didn't edit the Display and Version entries..

**Review by mivsvit — COMMENTED** [review]
DisplayVersion should be 5.15.3 (18551)

File: manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+- InstallerLocale: en-US
+  Architecture: arm64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://cdn.zoom.us/prod/5.15.3.18551/arm64/ZoomInstallerFull.msi
+  InstallerSha256: 301467491E6B93398374135F43B8114B271FE2348C1618211D010E42D900D5CE
+  ProductCode: '{543FD921-ABC5-4919-994E-F1CB8640C8B9}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Zoom(ARM64)
+    DisplayVersion:5.15.3 (18551)
```
Comment (BrandonWanHuanSheng) [inline manifests/z/Zoom/Zoom/5.15.3.18551/Zoom.Zoom.installer.yaml:97]:
```suggestion
    DisplayVersion: 5.15.3 (18551)

```

**Comment by BrandonWanHuanSheng** [comment]
Schema Error

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/112101

**Pull request closed without merging**